src/Field.php

```diff
@@ -143,6 +143,10 @@ public function __debugInfo()
             $arr['join'] = $this->join;
         }
 
+        if ($this->editable) {
+            $arr['editable'] = $this->editable;
+        }
+
```

Comment:
I don't think we need this. I try to keep only the essential information there. Join is quite impactful when you're debugging, but editable does not amount for much.

I suppose it's ok, for now but possibly will want to clean this up later.